notebooks/data-sources/TestGrid/metrics/build_pass_failure.ipynb

```diff
@@ -2,25 +2,25 @@
  "cells": [
```

Comment:
<p>What does it mean if a build has a value of "False" in both pass and fail column? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/172/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>It means a build contains a value in any of the other labels here: https://github.com/aicoe-aiops/ocp-ci-analysis/blob/1a3a7a58a340c401f8c68ef5d3e9c3f8e11ef864/docs/cell_labels_in_testgrid.md</p><p> </p><p>Although we never see occurences of most of the labels, 1, 4, 12, 13 are labels which are captured, so a false in both means its probably 4 or 13</p>